scalyr

```diff
@@ -663,6 +664,10 @@ def commandTimeseriesQuery(parser):
                         help='specifies the format in which numbers are emitted')
     parser.add_argument('--priority', choices=['high', 'low'], default='high',
                         help='specifies the execution priority for this query. Use low for scripted operations where a delay of a second or so is acceptable.')
+    parser.add_argument('--only-use-summaries', dest='onlyUseSummaries', action='store_true', default=False,
+                        help='specifies to query only summaries, and will not search the column store for any summaries not yet populated')
+    parser.add_argument('--no-create-summaries', dest='createSummaries', action='store_false', default=True,
```

Comment:
OK, this use of negatives is interesting and totally threw me.  Maybe this is normal for the standard python reader, but a comment would not go amiss (`# note we're inverting this (negative) param into args.createSummaries` or sth)

Reply:
Python argparser doesn't work well with boolean flags like this: `--create-summaries True -only-use-summaries False`... see [this explanation](https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse). 
Since our API defaults to `createSummaries = True` and `onlyUseSummaries = False`, I thought this was a good way to expose the new options. 

I will add a comment in the code as suggested.